python-examples/tests/test_get_concept_by_code_with_explicit_include_param.py

```diff
@@ -0,0 +1,53 @@
+import configparser
+import json
+import logging
+import os
+import requests
+import pytest
+from requests import Response
+
+
+@pytest.fixture(scope="module")
+def load_config():
+    """
+    Load configuration settings
+    """
+    # Create a ConfigParser object & read the file
+    config = configparser.ConfigParser()
+    config.read("config.ini")
+    return config
+
+
+class TestGetConceptWithIncludeParam:
+    """
+    Test case to get a concept by code with the include parameter. Make sure you have your authorization token by
+    running test_login.py
+    """
+    
+    # Create logger from pytest.ini settings
+    logger = logging.getLogger(__name__)
+    
+    def test_get_concept_by_code_with_include_param(self, load_config):
+        """
+        Test the get concept by code endpoint with SNOMEDCT terminology and a code for diabetes in the sandbox
+        project and an include parameter passed. This will call the termhub api and return the results
+        """
+        # SETUP
+        api_url: str = load_config.get("default", "url")
+        token: str = os.getenv("TOKEN")
+        terminology: str = "SNOMEDCT"
+        code: str = "73211009"
+        project_id: str = "sandbox"
+        params: dict[str, str] = {"inclue": "full"}
```

Comment:
looks like a typo.  should be "include".  please test that "full" and "minimal" give different results.

Reply:
fixed the typo. I'm also only copying the tests that are present in the curl-examples. If we want to add more tests, we should discuss in the next TH Standaup